client/src/ui/base/_themes.scss

```diff
@@ -207,6 +207,71 @@
   --baseline-limited-check: #1e8e3e;
   --baseline-limited-cross: #ea8600;
 
+  --cur-bg-color: #fcefe2;
```

Comment:
Nit: I'm not a big fan of the `cur` prefix, because it can be mistaken for "current". I think it would have been okay to use `curriculum-`.